src/lib/contants.ts

```diff
@@ -86,8 +86,11 @@ export const resourceSizeQueryParameters = {
   whole: "whole"
 } as const
 
+// todo: increment these when changing any of the tsx files since that will likely change any affected css, and built js hashes.
```

Comment:
Future builds.  Still thinking on this somewhat... I don't think you want it to auto-bump or anything since not every build is worthy of be a cache buster, but I also don't entirely love having to remember to do it either.  But It's ultimately not a massive deal since, like the comment says, you can always refresh the cache with a hard refresh.